database/buildSchema/06_trigger_queue.sql

```diff
@@ -40,8 +40,8 @@ CREATE OR REPLACE FUNCTION public.add_event_to_trigger_queue ()
     AS $trigger_queue$
 BEGIN
     INSERT INTO trigger_queue (trigger_type, "table", record_id, timestamp, status)
-        VALUES (NEW.trigger::public.trigger, TG_TABLE_NAME, NEW.id, CURRENT_TIMESTAMP, 'Triggered');
-    EXECUTE format('UPDATE %s SET trigger = ''Processing'' WHERE id = %s', TG_TABLE_NAME, NEW.id);
+        VALUES (NEW.trigger::public.trigger, TG_TABLE_NAME, NEW.id, CURRENT_TIMESTAMP, 'TRIGGERED');
+    EXECUTE format('UPDATE %s SET trigger = %L WHERE id = %s', TG_TABLE_NAME, 'PROCESSING', NEW.id);
```

Comment:
What is `%L`?

Reply:
string literal, I could have kept it as is, just capitalise, but the first time i changed it i remove the double `'` and broke it, so thought to just change to `%L`, %s doesn't work cause it needs to be quoted (unlike table name and id)